bucket_27/wtf/specification

```diff
@@ -0,0 +1,41 @@
+DEF[PORTVERSION]=	20180925
+DEF[DATADIR]=		{{PREFIX}}/share/wtf
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		wtf
+VERSION=		${PORTVERSION}
+KEYWORDS=		games
+VARIANTS=		standard
+SDESC[standard]=	Translates common Internet acronyms
+HOMEPAGE=		none
+CONTACT=		Leonid_Bobrov[mazocomp@disroot.org]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		https://downloads.sourceforge.net/project/bsdwtf/
+DISTFILE[1]=		wtf-${PORTVERSION}.tar.gz:main
+
+SPKGS[standard]=	single
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+LICENSE=		PUBDOM:single
+LICENSE_FILE=		PUBDOM:{{WRKDIR}}/LICENSE
+LICENSE_AWK=		PUBDOM:"^$"
```

Comment:
This probably should be:
```
PUBDOM:"^$$"
```
used the "env ENTERAFTER=extract ravenadm test wtf" command.  When you get the prompt, cd /construction/wtf and check the contents of LICENSE file to ensure they are correct.